runtime/oti/VMHelpers.hpp

```diff
@@ -2048,13 +2049,68 @@ class VM_VMHelpers
 #endif /* JAVA_SPEC_VERSION > 11 */
 	}
 
```

Comment:
A general comment about these Continuation specific helpers in this generic VMHelpers.hpp file....

Firstly, they should probably be in  ContinuationHelpers.cpp file (or *.hpp for inline purposes) .
Secondly, J9VMContinuation should really be a class and all these helpers and (whether from VMHelpers or from ContinuationHelpers) should be class member methods. Most of those methods have J9VMContinuation as a parameter (or if they don't they would be a static methods).

But I guess, there is  C style deeply rooted into most of VM structs, that I don't want to deal with (especially not as a part of these changes).